modules/standard/Set.chpl

```diff
@@ -143,12 +143,26 @@ module Set {
 
     // Returns true if the key was added to the hashtable.
     pragma "no doc"
-    proc _addElem(in elem: eltType): bool {
-      var (isFullSlot, idx) = _htb.findAvailableSlot(elem);
+    proc _addElem(pragma "no auto destroy" in elem: eltType): bool {
```

Comment:
It would be nice if we didn't need the pragma to do it. Could you make an issue about that (or something)?

Reply:
https://github.com/cray/chapel-private/issues/1796